src/compact_ed25519.h

```diff
@@ -25,6 +25,10 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#ifdef __cplusplus
```

Comment:
I'm fine with this, but lets then also update the `algamize.sh` script to not add another set of these `extern "C"` parts.

Reply:
Fixed, checked that algamized still builds. Here is a diff of what changes in the output:
https://gist.github.com/PetteriAimonen/9be61b2d1dca975ce98a0dcb04148d8e